packages/cozy-procedures/src/components/Overview.jsx

```diff
@@ -45,6 +45,26 @@ class Overview extends React.Component {
     if (job.state === 'done') {
       Alerter.success(t('overview.zip_ready'))
       client.collection('io.cozy.files').destroy(jsonFile)
+    } else if (job.state === 'errored') {
+      Alerter.error(t('overview.zip_errored'))
+      // if an error occured, destroy the zip and the json
+      client
+        .collection('io.cozy.files')
+        .find(
+          {
+            dir_id: job.message.dir_id,
+            name: job.message.filename
+          },
+          {
+            indexedFields: ['dir_id', 'name']
+          }
+        )
+        .then(resp => {
+          resp.data.forEach(zipFile => {
```

Comment:
can we have multiple zips? Or do you want to remove also the files within the zip? If the second, I don't think this is the wanted behavior. We just want to remove the json file?

Reply:
it's in case we have multiple zip (which should not happen)